WDL2CWL_example/varscan/samtoolsMpileup.cwl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env cwl-runner
+# This tool description was generated automatically by wdl2cwl ver. 0.2
+
+{
+    "baseCommand": [],
+    "outputs": [
+        {
+            "outputBinding": {
+                "glob": "$(inputs.sampleName).mpileup"
+            },
+            "type": "File",
+            "id": "rawMpileup"
+        }
+    ],
+    "inputs": [
+        {
+            "type": "File",
+            "id": "inputBAM"
+        },
+        {
+            "type": "File",
+            "id": "ReferenceGenome"
+        },
+        {
+            "type": "string",
+            "id": "sampleName"
+        }
+    ],
+    "requirements": [
+        {
+            "class": "ShellCommandRequirement"
+        },
+        {
+            "class": "InlineJavascriptRequirement"
+        }
+    ],
+    "arguments": [
+        {
+            "shellQuote": false,
+            "valueFrom": "samtools index $(inputs.inputBAM.path)samtools mpileup -B -f $(inputs.ReferenceGenome.path) $(inputs.inputBAM.path) > $(inputs.sampleName).mpileup"
```

Comment:
Did you test this? I think a `|` is missing.

`wdl2cwl` is a very old tool, I'm not sure if it ever worked.